client/lib/user/user.js

```diff
@@ -112,7 +117,6 @@ User.prototype.get = function() {
  * Fetch the current user from WordPress.com via the REST API
  * and stores it in local cache.
  *
- * @uses `wpcom`
```

Comment:
I'll have to look around the filesystem to see what to use instead of `@uses` to get the linter to pass.